src/pages/home/report/ReportActionsView.js

```diff
@@ -155,6 +161,13 @@ class ReportActionsView extends React.Component {
         }
     }
 
+    onMarkAsUnread(actionIndex) {
+        updateLastReadActionID(this.props.reportID, actionIndex, true);
+        this.setState({
+            unreadActionCount: this.props.report.maxSequenceNumber - actionIndex,
```

Comment:
The component should re-render with the correct `unreadActionCount` when we make the optimistic update?

Reply:
Yup. It does so. That makes sense, right?

Reply:
Oh sorry I was saying we should not need to set the state to trigger the re-render but looks like you've got it now thanks.